utilities/CojUtilities.ts

```diff
@@ -1,13 +1,14 @@
 import { COJ_EPOCH, GOLD_GUIDE_VERSION_REGEX } from "./Constants";
-
+import { DateUtilities } from "../utilities/DateUtilities";
 export function getStatusText(startDate: string | null) {
   if (!startDate) {
     return "Unknown status";
   }
-
+  
   return new Date(startDate) < COJ_EPOCH
     ? "Submitted directly to Local Office"
-    : "Not signed";
+    : DateUtilities.isWithin13Weeks(new Date(startDate)) ? "Not signed"
+    : "CoJ can only be signed within 13 weeks of the start date";

```

Comment:
Could we give them the date signing will be enabled, rather than have them work it out themselves?

e.g.
> Not signed, available from dd/mm/yyyy

We should avoid using "CoJ" in a user facing way, it's not a common acronym.